lib/govspeak.rb

```diff
@@ -155,6 +157,8 @@ def legislative_list_footnote_definitions(source)
 
         @footnote_definition_html << "</ol></div>"
       end
+
+      add_acronym_alt_text(@footnote_definition_html) if @acronyms.size.positive?
```

Comment:
Nice one 👏

Reply:
This has been updated to accommodate the use of `nil` in `@footnote_definition_html`, see [here](https://github.com/alphagov/govspeak/pull/216/files#diff-acb6db6cd67285bd5380398c9e51ced8940ed7c4110706d038cbb243310e4055R164-R166)